tests/BlazingSQLTest/Runner/runTest.py

```diff
@@ -1404,7 +1404,7 @@ def run_query(
 
     algebra = kwargs.get("algebra", "")
 
-    comparing = kwargs.get("comparing", "false")
+    comparing = kwargs.get("comparing", "true")
```

Comment:
what is this doing?

Reply:
This parameter was introduced in the get `CURRENT_DATE/TIMESTAMP` feature,  due that the results could be different. And since there,  some tests were "succeeding" wrongly.